app/src/main/java/org/p2p/wallet/jupiter/repository/model/JupiterSwapToken.kt

```diff
@@ -11,4 +12,10 @@ data class JupiterSwapToken(
     val tokenName: String,
     val tokenSymbol: String,
     val tags: List<String>
-)
+) {
+    fun isSol(): Boolean = tokenMint.base58Value == Constants.WRAPPED_SOL_MINT
+}
+
+fun List<JupiterSwapToken>.findTokenByMint(mint: Base58String): JupiterSwapToken? {
```

Comment:
why you did not use firstOrNull { }  ?

Reply:
it was used previously in mapper only, this signature makes it more usable then just firstOfNull